modules/api/pages/activities-tasks-flow-nodes-api.adoc

```diff
@@ -582,11 +582,13 @@ xref:ROOT:rest-api-overview.adoc#resource_search[Standard search parameters] are
  ** o: name, priority, dueDate, state, processDefinitionId, processInstanceId,
 parentActivityInstanceId, assigneeId, parentContainerId, displayName, reachedStateDate
  ** s: search on any field that can be used to order results
- ** f: assigned_id, state, name, displayName, processDefinitionId, caseId, parentCaseId, rootCaseId
+ ** f: assigned_id, user_id, show_assigned_to_others, state, name, displayName, processDefinitionId, caseId, parentCaseId, rootCaseId
```

Comment:
`user_id` should be part of a PR on an older branch.

Reply:
which one? :D

Reply:
2021.2

Reply:
ok I will add new PR for that

Reply:
do we need to do the same for `2021.2` and `2022.1 `?

Reply:
https://github.com/bonitasoft/bonita-doc/pull/2091

Reply:
No, it will be merged